<!-- https://www.cnblogs.com/hong2016/p/7624280.html -->
<launch>

    <arg name="record" default="false"/>

    <!-- Important: Use simulated time published on /clock topic, otherwise TF will be broken -->
    <!-- <param name="/use_sim_time" value="true"/> -->

    <node name="cloud_detection" pkg="pcl" type="cloud_detection_node" 
         output="screen">
        <rosparam command="load" file="$(find pcl)/config/bluerov_zed_parameters.yaml"/>
        <!-- <rosparam command="load" file="$(find pcl)/config/uuv_simulator_parameters.yaml"/> -->
    </node>

    <!-- http://wiki.ros.org/rosbag/Commandline
    http://wiki.ros.org/rosbag/Tutorials -->
    <group if="$(arg record)">
        <node name="record" pkg="rosbag" type="record"
            args="-o cloud /zed/zed_node/point_cloud/cloud_registered /tf /tf_static /clock /zed/zed_node/left/image_rect_color">
        </node>
    </group>

</launch>